css-three-column-layout/styles.css

```diff
@@ -0,0 +1,77 @@
+body {
+  background-color: rgb(25, 150, 218);
+  font-family: Lato, sans-serif;
+  height: 100vh;
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+h1 {
+  font-size: 2rem;
+  color: white;
+}
+
+h2 {
+  text-align: center;
+  white-space: nowrap;
+  overflow: hidden;
+  color: white;
+  font-size: 1.27rem;
+  margin-bottom: 0;
+}
+
+p {
+  text-align: center;
+  font-size: 0.9rem;
+  color: lightgray;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 60rem; /* Set a maximum width to center the content */
+}
+
+.row {
+  justify-content: center; /* Center the h1 element horizontally */
+  display: flex;
+  width: 100%; /* Each row takes full container width */
+}
+
+.column-one-third {
+  flex-basis: calc(
+    100% * 1 / 3
+  ); /* Each column takes one-third of the row width with a little spacing between them */
+
+  margin: 13px; /* Add some spacing between columns */
+  margin-left: 2rem;
+  margin-right: 2rem;
```

Comment:
Lines 50 & 51 are overriding what you'd be applying on line 49. You should just take advantage of the short hand and not write out `margin-left` & `margin-right` .

Reply:
Updated